core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -70,8 +77,10 @@ interface ActiveSuggestionDict {
   'target_type': string;
 }
 
+// This is used to display the details of the active contribution in the modal.
+// The details are null if there is no active contribution.
 interface ActiveContributionDict {
-  'details': ActiveContributionDetailsDict;
+  'details': ActiveContributionDetailsDict | null;
```

Comment:
I don't understand the comment justifying this. If there is no active contribution then why is the modal able to be shown at all?

Reply:
No, in comment i mean after deletion of all contribution we have no active contribution. And correct after we have no contribution we need to close model, doing same with the help of null. That's how null is needed here.

Reply:
may be null when suggestion's corresponding opportunity is deleted. 
like issue [#14234](https://github.com/oppia/oppia/issues/14234) give more insight.

Reply:
Ah. I think then you can put "See issue #14234" after the comment on line 74. Otherwise other people looking at the code won't understand what is going on conceptually.

Reply:
Done